### ISSUE TITLE
getHostedPaymentPageController response is always None

### ISSUE THREAD

**Issue by felixSchl** [AuthorizeNet/sdk-python#83]
The code below is copied [almost verbatim](https://github.com/AuthorizeNet/sample-code-python/blob/18cd0505a6b049bff889d8b9b097ece4c6c29176/PaymentTransactions/get-an-accept-payment-page.py) from the python example.

For some reason `paymentPageResponse` below always returns `None`. The only way I can get it to at least a failure is by switching from `constants.SANDBOX` to `constants.PRODUCTION`.

What is the meaning of `None`? The API key and transaction key I provide are for a sandbox account I created earlier today. Do I have to enable anything in the merchant interface for this to work?

```python
from authorizenet import apicontractsv1
from authorizenet.apicontrollers import *
from authorizenet.constants import constants

def test():
            merchantAuth = apicontractsv1.merchantAuthenticationType()

            merchantAuth.name = <API_LOGIN_ID>
            merchantAuth.transactionKey = <TRANSACTION_KEY>

            setting1 = apicontractsv1.settingType()
            setting1.settingName = apicontractsv1.settingNameEnum.hostedPaymentButtonOptions
            setting1.settingValue = "{\"text\": \"Pay\"}"

            setting2 = apicontractsv1.settingType()
            setting2.settingName = apicontractsv1.settingNameEnum.hostedPaymentOrderOptions
            setting2.settingValue = "{\"show\": false}"

            settings = apicontractsv1.ArrayOfSetting()
            settings.setting.append(setting1)
            settings.setting.append(setting2)

            transactionrequest = apicontractsv1.transactionRequestType()
            transactionrequest.transactionType = "authCaptureTransaction"
            transactionrequest.amount = Decimal ('100.00')

            paymentPageRequest = apicontractsv1.getHostedPaymentPageRequest()
            paymentPageRequest.merchantAuthentication = merchantAuth
            paymentPageRequest.transactionRequest = transactionrequest
            paymentPageRequest.hostedPaymentSettings = settings

            paymentPageController = getHostedPaymentPageController(paymentPageRequest)
            paymentPageController.setenvironment(constants.SANDBOX)
            paymentPageController.execute()
            paymentPageResponse = paymentPageController.getresponse()

test()
```

Edit: And also is there a way to enable HTTP level logging. It's one black box atm

**Comment by felixSchl** [comment]
Alright, so with `logging` turned on, I can see:

```
ERROR:root:Error retrieving http response from: https://apitest.authorize.net/xml/v1/request.api for request: <xml.dom.minidom.Document instance at 0x10e947878>
```

**Comment by felixSchl** [comment]
Apparently fixed by updating to python 2.7.13 (from 2.7.10). I think it was due to an old OpenSSL version. Would still be nice if this library was vocal about what actually happened rather than just returning `None`. I'll leave this one here, but hopefully someone else finds use in it.
